File: CPP/Aksesoris.cpp
#include "PetShop.cpp"


class Aksesoris : public PetShop
{
private:
    string jenis;
    string bahan;
    string warna;

public:
    Aksesoris()
    {
    }

    Aksesoris(int id, string namaProduk, int hargaProduk, int stokProduk, string jenis, string bahan, string warna)
    {
        setId(id);  // Pakai setter
        setNamaProduk(namaProduk);
        setHargaProduk(hargaProduk);
        setStokProduk(stokProduk);
        this->jenis = jenis;
        this->bahan = bahan;
        this->warna = warna;
    }

    string getJenis()
    {
        return jenis;
    }

    void setJenis(string jenis)
    {
        this->jenis = jenis;
    }

    string getBahan()
    {
        return bahan;
    }

    void setBahan(string bahan)
    {
        this->bahan = bahan;
    }

    string getWarna()
    {
        return warna;
    }

    void setWarna(string warna)
    {
        this->warna = warna;
    }

    // Destructor
    ~Aksesoris()
    {
    }

};

File: Python/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    def __init__(this):
        this.jenis = ""
        this.bahan = ""
        this.warna = ""
        
    # Setter & Getter
    def get_jenis(this):
        return this.jenis
    
    def set_jenis(this, jenis):
        this.jenis = jenis
        
    def get_bahan(this):
        return this.bahan
    
    def set_bahan(this, bahan):
        this.bahan = bahan
        
    def get_warna(this):
        return this.warna
    
    def set_warna(this, warna):
        this.warna = warna